app/assets/stylesheets/expressive.scss

```diff
@@ -0,0 +1,20 @@
+.flex {
+  display: flex;
+}
+
+.flex-column {
+  display: flex;
+  flex-direction: column;
+}
+
+.full-width {
+  width: 100wv;
+}
+
+.flex-wrap {
+  flex-wrap: wrap;
+}
+
+.margin-20 {
```

Comment:
I agree that it's also a valid way to do it, but I think we have to be careful when trying expressive CSS to avoid repeating a lot of rules and making it difficult to read. Even in this style of writing code, I think you can express ideas in a more semantic way and then mixing them to achieve the visual result you want. Maybe you have a centered-items-container which only defines the distribution of the items (using flex, for example) and then you add other classes to define the colors combination (background color for the container and color for the text inside) and then another class to change the size of the text in that specific container. That is still expressive and I think it makes more sense to reuse those kind of things that making such granular classes as defining specific margins every time when they probably will be the same in different views or distributions. Remember that mixins will probably be useful when writing expressive CSS, because you can define several similar classes by including a generic mixin (for example for flex, you could do a mixin that receives the direction, the main axis alignment and the secondary axis alignment so you just include the mixing with the desired values instead of writing the properties a lot of times). This [https://ferdychristant.com/in-defense-of-expressive-css-5d44862d5c56](post) talks about this and a way to implement it, I think it's nice to read! Anyway, it's a matter of taste as well, it is difficult to set an standard, I'm just giving my opinion here :stuck_out_tongue: